src/add-to-project.ts

```diff
@@ -50,11 +50,16 @@ export async function addToProject(): Promise<void> {
       core.info(`Skipping issue ${issue?.number} because it doesn't match all the labels: ${labeled.join(', ')}`)
       return
     }
-  } else {
+  } else if (labelOperator === 'or') {
     if (labeled.length > 0 && !issueLabels.some(l => labeled.includes(l))) {
       core.info(`Skipping issue ${issue?.number} because it does not have one of the labels: ${labeled.join(', ')}`)
       return
     }
+  } else if (labelOperator === 'not') {
```

Comment:
I think we need to handle this prior to the `or` case, and leave the `or` case in an else statement, so that the default `or` gets applied in the event that we don't have another matching `labelOperator`


I think we should probably validate these cases and throw so that we can be explicit about them all, but I think that's out of the realm of this pr